### PR TITLE
feat(concept):concept rcms added

### PR DESCRIPTION
**Pull request by aysuakbaba**

### CODE REVIEW

File: server/controllers/concept.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+exports.findByRoadmapId = (req, res) => {
+  Concept.findByRoadmapId(req.params.roadmap_id, (err, data) => {
+    if (err) {
+      if (err.kind === 'not_found') {
+        res.status(400).send({
+          message: `There is no concept with roadmap id ${req.params.roadmap_id}`
+        });
+        return;
+      }
+      console.log('aysu');
```
Comment (kaganecee) [inline server/controllers/concept.js:51]: don't forget logs please. Examine your commits before pushing.

File: server/routes/concept.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const concept = require('../controllers/concept.js');
+const auth = require('../middleware/auth');
+
+const router = express.Router();
+
+router.route('/concept').post(auth, concept.create);
+router.route('/concept/:id').get(auth, concept.findById).delete(auth, concept.delete);
+router.route('/concept/getByRoadmap/:roadmap_id').get(auth, concept.findByRoadmapId);
```
Comment (kaganecee) [inline server/routes/concept.js:9]: we can set our route as a concept/roadmap/:id . What do u think?

**Review by kaganecee — CHANGES_REQUESTED** [review]

**Comment by kaganecee** [comment]
Also please rename your commit message. Add space after the project name brackets

File: server/controllers/concept.js (hunk 1 above)
Comment (pakyardim) [reply server/controllers/concept.js:51]: an easy but honest mistake to make.

File: server/routes/concept.js (hunk 2 above)
Comment (pakyardim) [reply server/routes/concept.js:9]: if you mean the `concept/getByRoadmap/:roadmap_id` endpoint, I think changing it to `concept/roadmap/:id` would be more appropriate.

**Review by pakyardim — COMMENTED** [review]
Despite the minor mistake of forgetting to delete the unnecessary `console.log`, I find your coding approach inspiring.

**Pull request closed without merging**